Resources/views/Form/datepicker.html.twig

```diff
@@ -39,12 +39,14 @@ file that was distributed with this source code.
 
 {% block sonata_type_datetime_picker_widget_html %}
     {% if datepicker_use_button %}
-        <div class='input-group date' id='dtp_{{ id }}'>
+        <div class='input-group date {% if not dp_options['pickDate'] %}timepicker{% endif %}' id='dtp_{{ id }}'>
```

Comment:
@greg0ire WDYT of `if dp_options['pickDate'] is defined and not dp_options['pickDate']` for BC?

Reply:
would be better indeed